go/wg.go

```diff
@@ -22,21 +22,20 @@ package athenadriver
 
 import (
 	"context"
-
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/service/athena"
-	"github.com/aws/aws-sdk-go/service/athena/athenaiface"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	v2athena "github.com/aws/aws-sdk-go-v2/service/athena"
+	v2athenatypes "github.com/aws/aws-sdk-go-v2/service/athena/types"
```

Comment:
Is there a reason for prefixing these with `v2`?

Reply:
Just leftover from the migration process I've been following - fixed!